.github/workflows/iroha2-ci-image.yml

```diff
@@ -1,6 +1,11 @@
 name: I2::CI::Publish
```

Comment:
```suggestion
name: I2::Release::Publish
```

Reply:
NP, I can rename these to whatever you like. I just warn if it will be informative. Because `I2::Publish` job is not so differed from `I2::Dev::Publish` or `I2::Release::Publish` and perhaps someone might mix it with the main iroha image build and push. Also, with this workflow, we can build & push not only profiling image, but image based on glibc without `--profile profiling ` as well.
But it's just my opinion that I would to convince.

Reply:
You are correct. Don't change this